dev/benchmarks/metrics_center/test/skiaperf_test.dart

```diff
@@ -262,4 +274,166 @@ void main() {
       throwsA(anything),
     );
   });
+
+  test('SkiaPerfGcsAdaptor computes name correctly', () async {
+    expect(
+      await SkiaPerfGcsAdaptor.comptueObjectName(
```

Comment:
The test seems based on real github http requests, flakiness maybe expected?

Reply:
True. Let me mock it to avoid any network flakiness.